src/dombeck_lab_to_nwb/azcorra2023/interfaces/picoscope_eventinterface.py

```diff
@@ -33,8 +35,17 @@ def __init__(self, folder_path: FolderPathType, channel_ids: Optional[list] = No
     def get_metadata(self) -> dict:
         metadata = super().get_metadata()
         metadata["Events"] = dict(
-            name="PicoscopeEvents",
-            description="Contains the onset times of binary signals from the PicoScope.",
+            EventTypesTable=dict(
+                name="PicosScopeEventTypes",
+                description="Contains the type of binary signals from PicoScope.",
+            ),
+            EventsTable=dict(
+                name="PicoscopeEvents",
+                description="Contains the onset times of binary signals from PicoScope.",
+            ),
+            TtlsTable=dict(
+                description="Contains the 405 nm and 470 nm illumination onset times.",
+            ),
         )
```

Comment:
you don't put the respective TtlTypesTable dict because you reuse the description for TtlsTable, correct?

Reply:
Yes